src/middlewared/middlewared/plugins/pool.py

```diff
@@ -55,3 +59,44 @@ class PoolService(CRUDService):
             yield
         async for i in await self.middleware.call('zfs.pool.get_disks', pool['name']):
             yield i
+
+    @private
+    def configure_resilver_priority(self):
+        """
+        Configure resilver priority based on user selected off-peak hours.
+        """
+        resilver = self.middleware.call_sync('datastore.config', 'storage.resilver')
+
+        if not resilver['enabled']:
+            return
+
+        higher_prio = False
+        weekdays = map(lambda x: int(x), resilver['weekday'].split(','))
```

Comment:
This can result in an error if a user actually enters '' (i.e. blank) in the resilver['weekday']. Why do we need that blank to be true, just asking?